<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pGetCandidatePage</class>
 <widget class="QWizardPage" name="pGetCandidatePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>614</width>
    <height>181</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Extract primer candidates from target sequences or load user's primer candidates:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="pixmap">
           <pixmap resource="PRISE.qrc">:/new/prefix1/images/exclamation.bmp</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Note:   If you load your own primer candidates, the program will not
consider primer candidates extracted from target sequences</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton_designByInput">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Design primers based on target and non-target sequences</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="radioButton_loadCands">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Load user's primer candidates</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_CandFilename"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_browseCandFile">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
     <zorder></zorder>
     <zorder>horizontalSpacer</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="PRISE.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_browseCandFile</sender>
   <signal>clicked()</signal>
   <receiver>pGetCandidatePage</receiver>
   <slot>browseCandidatesFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseCandidatesFile()</slot>
 </slots>
</ui>
